File: app/src/main/java/cvMaker/resumeMaker/cvBuilder/resumeBuilder/cvTemplate/createCv/freeCv/roomDatabaseClasses/model/ProjectsEntity.java
package cvMaker.resumeMaker.cvBuilder.resumeBuilder.cvTemplate.createCv.freeCv.roomDatabaseClasses.model;

import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

@Entity(tableName = "projectTable")
public class ProjectsEntity {

    @PrimaryKey(autoGenerate = true)
    int projectID;

    String projectTitle = "";

    String userId = "";

    @Ignore
    public ProjectsEntity() {
    }

    @Ignore
    public ProjectsEntity(String projectTitle, String userId) {
        this.projectTitle = projectTitle;
        this.userId = userId;
    }

    public ProjectsEntity(int projectID, String projectTitle, String userId) {
        this.projectID = projectID;
        this.projectTitle = projectTitle;
        this.userId = userId;
    }

    public int getProjectID() {
        return projectID;
    }

    public void setProjectID(int projectID) {
        this.projectID = projectID;
    }

    public String getProjectTitle() {
        return projectTitle;
    }

    public void setProjectTitle(String projectTitle) {
        this.projectTitle = projectTitle;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }
}

File: app/src/main/java/cvMaker/resumeMaker/cvBuilder/resumeBuilder/cvTemplate/createCv/freeCv/roomDatabaseClasses/model/ProjectsEntity.kt
package cvMaker.resumeMaker.cvBuilder.resumeBuilder.cvTemplate.createCv.freeCv.roomDatabaseClasses.model

import androidx.room.Entity
import androidx.room.Ignore
import androidx.room.PrimaryKey

@Entity(tableName = "projectTable")
class ProjectsEntity {
    @PrimaryKey(autoGenerate = true)
    var projectID = 0
    var projectTitle = ""
    var userId = ""

    @Ignore
    constructor() {
    }

    @Ignore
    constructor(projectTitle: String, userId: String) {
        this.projectTitle = projectTitle
        this.userId = userId
    }

    constructor(projectID: Int, projectTitle: String, userId: String) {
        this.projectID = projectID
        this.projectTitle = projectTitle
        this.userId = userId
    }
}